api/HTMLMediaElement.json

```
@@ -2792,7 +2792,9 @@
             "chrome": {
               "version_added": "49"
             },
-            "chrome_android": "mirror",
+            "chrome_android": {
+              "version_added": false
```

Like the related `setSinkId` feature (see above in this file), we could add a the same comment here:
```suggestion
              "version_added": false,
              "notes": "Not available due to <a href='https://crbug.com/648286'>a limitation in Android</a>."
```